bin/mmdblookup.c

```diff
@@ -79,9 +79,9 @@ int wmain(int argc, wchar_t **wargv)
                     GetLastError());
             exit(1);
         }
-        utf8_string = malloc(utf8_width);
+        utf8_string = calloc(1, utf8_width);
```

Comment:
Would this one make more sense with the arguments reversed? Maybe with a sizeof(char)?

Reply:
Thanks! I force-pushed a fix.